### DISCUSSION TITLE
Sentry tagging not working on Composer

### DISCUSSION THREAD

**Discussion by nirajdewani** [apache/airflow#27595] [General]
### Apache Airflow version

Other Airflow 2 version (please specify below)

### What happened

Composer version: 2.0.10
Airflow version: 2.2.3

Airflow comes with Sentry integration, as documented [here](https://airflow.apache.org/docs/apache-airflow/1.10.15/errors.html?highlight=sentry).
With `SENTRY_DSN` configured as an environment variable, we generated a (run-time) error in a DAG. The corresponding exception showed up as a Sentry error (as expected). Though the error was missing tags listed [here](https://airflow.apache.org/docs/apache-airflow/1.10.15/errors.html?highlight=sentry#tags).

### What you think should happen instead

In addition to the tags in the screenshot below, we expect to see the tags listed [here](https://airflow.apache.org/docs/apache-airflow/1.10.15/errors.html?highlight=sentry#tags).
![Screen Shot 2022-11-03 at 4 29 42 PM](https://user-images.githubusercontent.com/10428678/199854218-3bd598cd-6f1a-40b5-87f0-8593413b481b.png)

### How to reproduce

1. Create a Sentry project (using either Sentry's self-hosted [option](https://develop.sentry.dev/self-hosted/) or Sentry's SaaS–here's a [free trial](https://sentry.io/signup/))
2. Create a 2.0.10 versioned Composer instance with Airflow version 2.2.3, configured with `SENTRY_DSN` from the above project
3. Create a DAG that results in a runtime error
4. Run the DAG
5. Check Sentry for the corresponding error–check tags on the error

### Operating System

Whatever Composer comes with.

### Versions of Apache Airflow Providers

apache-airflow==2.2.3+composer

### Deployment

Composer

### Deployment details

Composer version: 2.0.10
Airflow version: 2.2.3

### Anything else

All the errors we see in Sentry do not have the necessary tags. The right tags would make it easier to browse errors, and understand their source.

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by eladkal** [comment]
The reproduce steps you show are on Composer.
If this is Composer issue please submit a bug to composer.

If the issue is repreducable on Airflow latest version please edit the report accordingly.

**Comment by potiuk** [comment]
Converted to discussion - no response from user, but seemss to be targeted to composer.
